lib/buffer.js

```diff
@@ -237,6 +237,17 @@ Buffer.from = function from(value, encodingOrOffset, length) {
   );
 };
 
+// Identical to the built-in %TypedArray%.of(), but avoids using the deprecated
+// Buffer() constructor.
+//
+// Refs: https://tc39.github.io/ecma262/#sec-%typedarray%.of
+Buffer.of = (...items) => {
+  const newObj = createUnsafeBuffer(items.length);
+  for (var k = 0; k < items.length; k++)
+    newObj[k] = items[k];
+  return newObj;
```

Comment:
Line-to-line code duplication of `fromArrayLike`.

Reply:
No. `fromArrayLike` uses `allocate()` while this function uses `createUnsafeBuffer()`. The former may be pooled, which is [not desirable](https://github.com/nodejs/node/pull/19682#discussion_r178172872).